### ISSUE TITLE
Моля за помощ

### ISSUE THREAD

**Issue by lamenco** [luchob/softuni-feb2023#135]
Здравей, осмелявам се да ти пиша може би за глупост, но просто правя задача за интервю и не виждам явно елементарна грешка. Ако намериш малко време би ли отделил да погледнеш кода --> https://github.com/lamenco/CarCatalog/tree/master/src

Не знам защо като пусна GET заявка за всички коли брандовете седят на NULL при положения, че като дебъгвам се сейфат както трябва и от там ми се чупи търсачка да търся по бранд.

Благодаря!

**Comment by luchob** [comment]
Привет! 

Соколкото виждам при мен...

<img width="1020" alt="Screenshot 2023-05-18 at 20 32 19" src="https://github.com/luchob/softuni-feb2023/assets/10339738/0033375e-25c2-49e5-8b7f-dc76279ca6fc">

**Comment by lamenco** [comment]
Да, то задачата дето ми я дадоха малко е объркваща и не ги сетвам брандовете, защото не са реални.
Когато запазвам Car и сетвам Бранд към Модел и после Модел към Car не разбирам защо не се запазва Бранда във връзката към Car.

**Comment by luchob** [comment]
_Когато запазвам Car и сетвам Бранд към Модел и после Модел към Car не разбирам защо не се запазва Бранда във връзката към Car._

Анотацията `ManyToOne` има пропърти `CascadeType`,  което определя поведението на хибернейт когато запазваш някаква релация. By default е - не прави нищо.
